src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -385,6 +386,18 @@ TEST_P(OVClassSetDevicePriorityConfigTest, SetConfigAndCheckGetConfigNoThrow) {
     ASSERT_EQ(devicePriority, configuration[ov::device::priorities.name()].as<std::string>());
 }
 
+TEST_P(OVClassSetTBBForceTerminatePropertyTest, SetConfigNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+
+    bool value = true;
+    OV_ASSERT_NO_THROW(ie.set_property(deviceName, ov::force_tbb_terminate(false)));
```

Comment:
force_tbb_terminate is property of OV runtime. Please remove device name in test case.